sequoia/settings/rl/continual/setting.py

```diff
@@ -1184,9 +1188,9 @@ def _make_env_dataloader(
             # Seed each environment with its own seed (based on the base seed).
             env.seed([seed + i for i in range(env_dataloader.num_envs)])
         else:
-            env.seed(seed)
```

Comment:
Not ideal. If `seed` being None causes an error, then it would be better to just raise an error here if `seed` is None, and then perhaps give it a default value of `123`.